cloudflare/resource_cloudflare_split_tunnel_exclude.go

```diff
@@ -0,0 +1,127 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudflareSplitTunnelExclude() *schema.Resource {
+	return &schema.Resource{
+		Read:   resourceCloudflareSplitTunnelsExcludeRead,
+		Update: resourceCloudflareSplitTunnelsExcludeUpdate,
```

Comment:
we'll also need to include the `Create` here (even if it reuses the `Update` method) otherwise the resource won't be created in the API or state.

Reply:
looking further down at the code, we may want to have a separate method 🤔 it looks like your update fetches all the existing values and sends them in the payload too which may not be desirable for a new resource given there will only be _new_ values to send.

Reply:
I was under the impression that `Create` matched up with a `POST` request and we don't have a `POST` request for split tunnels in the API. Am I wrong on that?